dev/com.ibm.ws.io.smallrye.reactive.messaging/src/io/smallrye/reactive/messaging/extension/ReactiveMessagingExtension.java

```diff
@@ -0,0 +1,116 @@
+package io.smallrye.reactive.messaging.extension;
```

Comment:
Apache 2 need - attribution and notices obligation observed if you create/ship a derivative work

Reply:
done